index.js

```diff
@@ -81,13 +80,18 @@ class ClinicDoctor extends events.EventEmitter {
       env: Object.assign({}, process.env, customEnv)
     })
 
-    if (this.detectPort) {
-      proc.stdio[3].once('data', (data) => {
+    // Use fd 3 for both Port and Source Warning
+    proc.stdio[3].once('data', (data) => {
+      if (this.detectPort) {
         this.emit('port', Number(data), proc, () => {
           proc.stdio[3].destroy()
         })
-      })
-    }
+      }
+      if (data.toString() === 'source_warning') {
+        this.emit('warning', 'Transpiled code is not supported')
```

Comment:
Why is it not supported really? Is there anything code specific in this one?

Reply:
@mcollina - Not 100% sure. @goto-bus-stop might have a better idea on it.

Reply:
@mcollina - Closing this. Looks like we only need this change for Flame and Bubbleprof.